From linux-serial-owner@vger.kernel.org  Wed Oct  6 13:16:43 2010
From: Breno Leitao <leitao@linux.vnet.ibm.com>
To: greg@kroah.com
Cc: Scott.Kilau@digi.com, linux-kernel@vger.kernel.org,
	linux-serial@vger.kernel.org,
	Breno Leitao <leitao@linux.vnet.ibm.com>
Subject: tty: MAINTAINERS: add drivers/serial/jsm/ as maintained driver
Date: Wed,  6 Oct 2010 13:04:48 -0300
Message-Id: <1286381088-7614-1-git-send-email-leitao@linux.vnet.ibm.com>

Soott Kilau is handing off the maintainership of the jsm serial
driver to me, and this patch just add the driver as maintained.

Signed-off-by: Breno Leitao <leitao@linux.vnet.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -3311,6 +3311,12 @@ F:	fs/jbd*/
 F:	include/linux/ext*jbd*.h
 F:	include/linux/jbd*.h
 
+JSM Neo PCI based serial card
+M:	Breno Leitao <leitao@linux.vnet.ibm.com>
+L:	linux-serial@vger.kernel.org
+S:	Maintained
+F:	drivers/serial/jsm/
+
 K8TEMP HARDWARE MONITORING DRIVER
 M:	Rudolf Marek <r.marek@assembler.cz>
 L:	lm-sensors@lm-sensors.org
